ThePensionsRegulator.Frontend/Views/Shared/TPR/TPRHeaderLockup.cshtml

```diff
@@ -29,6 +30,36 @@
                 {
                     <tpr-header-search action="@Model.HeaderSearchActionPath" autocomplete-url="@Model.HeaderSearchAutocompleteUrl" placeholder="@Model.HeaderSearchPlaceholderText" aria-label="@Model.HeaderSearchAriaLabel" input-name="@Model.HeaderSearchInputName"></tpr-header-search>
                 }
+                @if (Model.ShowHeaderMenu)
+                {
+                    <tpr-header-menu>
+                        @if (Model.HeaderMenuParentItems != null)
+                        {
+                            @foreach (var item in Model.HeaderMenuParentItems)
+                            {
+                                <tpr-header-menu-parent-item href="@item?.LinkDestination" link-text="@item?.LinkText">
```

Comment:
We shouldn't render a `<tpr-header-menu-parent-item>` if `item` is null. Ideally `item` would never be null, so explore whether you can make `Model.HeaderMenuParentItems` a `List<TprHeaderMenuParentItem>` rather than `List<TprHeaderMenuParentItem>?`. If not then use `var item in Model.HeaderMenuParentItems.OfType<TprHeaderMenuParentItem>()` in the foreach statement to eliminate any nulls.